systers_portal/meetup/forms.py

```diff
@@ -6,11 +6,32 @@
 
 from common.forms import ModelFormWithHelper
 from common.helpers import SubmitCancelFormHelper
-from meetup.models import Meetup, MeetupLocation, Rsvp, SupportRequest
+from meetup.models import Meetup, MeetupLocation, Rsvp, SupportRequest, RequestMeetupLocation
 from users.models import SystersUser
 from common.models import Comment
 
 
+class RequestMeetupLocationForm(ModelFormWithHelper):
+    """ Form to create a new Request Meetup Location by a systers user. """
+    class Meta:
+        model = RequestMeetupLocation
+        fields = ('name', 'slug', 'location', 'description', 'email')
+        helper_class = SubmitCancelFormHelper
+        helper_cancel_href = "{% url 'list_meetup_location' %}"
+
+    def __init__(self, *args, **kwargs):
+        self.user = kwargs.pop('user')
+        super(RequestMeetupLocationForm, self).__init__(*args, **kwargs)
+
+    def save(self, commit=True):
+        """Override save to add requestor to the instance"""
+        instance = super(RequestMeetupLocationForm, self).save(commit=False)
+        instance.user = SystersUser.objects.get(user=self.user)
+        if commit:
+            instance.save()
+        return instance
+
```

Comment:
Again here also we would be having the same issue and you would want to properly handle instances in the case when the admin would approve or reject a new meetup location request. Does this work for you, were you able to approve a request after login as admin?

Reply:
@tapasweni-pathak I took care of the issue. Updated my PR. Thanks.